packages/admin/src/table/filterbar/FilterBarActiveFilterBadge.tsx

```diff
@@ -0,0 +1,62 @@
+import { createStyles, Theme, Typography, WithStyles, withStyles } from "@material-ui/core";
+import * as React from "react";
+
+export type CometAdminFilterBarActiveFilterBadgeClassKeys = "hasValueCount";
+
+export const dirtyFieldsCount = (values: Record<string, any>, results: string[] = []) => {
+    const r = results;
+
+    Object.keys(values).forEach((key) => {
+        const value = values[key];
+        if (typeof value !== "object") {
+            if (Array.isArray(value)) {
+                r.push(...value);
+            } else {
+                console.log("singleValue");
+                r.push(value);
+            }
+        } else if (typeof value === "object") {
+            dirtyFieldsCount(value, r);
+        }
+    });
+
+    return r.length;
```

Comment:
why not count like you did before? This api with results as parameter is strange (yes, it's even part of the public api)

Reply:
i refactored my method, now its just using a count variable and not an results array :)

Reply:
please remove that count argument, no reason for that

Reply:
and this magic line isn't the best we can do:
`} else if ("min" in value && "max" in value) {`
(btw: also commit the story you used for testing this!)

At least we must make sure only min and max exist (and no other keys).

Or we find a different solution (like different count based on fields?)

Reply:
do i declare de variable above the function then because i need a variable which gets increased by the number of changed fieldValues ? :) or how do i do that in a recursive function otherwise ? 

and i am getting out of ideas how i should implement that task ? getting lost :O